File: C#/owoify.cs
using System;
using System.Text.RegularExpressions;

namespace owoify
{
    public class owo
    {
        /// <summary>
        /// owoify a given string
        /// </summary>
        /// <param name="String">String to owoify</param>
        /// <returns>
        /// owoified String
        /// </returns>
        public static string owoify(string String)
        {
            if (String == null) throw new ArgumentNullException(nameof(String));

            var str = String;
            Random rnd = new Random();
            string[] prefix =  {
            "OWO", "OwO", "0w0", "< 3", "UwU", "HIIII!", "H - hewwo??", "Huohhhh."
            };
            string[] suffix =
            {
                "..,", "fwendo", "Huoh.", "._.", ";-;", ";_;", "(；ω；)", "ÙωÙ", "UwU", "(人◕ω◕)", "(●´ω｀●)", "(✿ ♡‿♡)",
                "(◠‿◠✿)", "^ - ^", "^ _ ^", "> _ <", "> _ >", ":P", ":3", ";3", "x3", ":D", "xD", "XDDD", "（＾ｖ＾）",
                "ㅇㅅㅇ", "(• o •)", "ʕ•̫͡•ʔ", "ʕʘ‿ʘʔ", "(　'◟ ')"
            };

            str = str.Replace("l", "w")
                .Replace("r", "w")
                .Replace("L", "W")
                .Replace("R", "W");
            str = Regex.Replace(str, "ha(ve|s)", "haz", RegexOptions.IgnoreCase);
            str = Regex.Replace(str, "n[o]+", 
                m => string.Format(m.Groups[0].Value.Replace("o", "u")), RegexOptions.IgnoreCase);
            str = Regex.Replace(str, "your", "ur", RegexOptions.IgnoreCase);
            str = Regex.Replace(str, "you", "uu", RegexOptions.IgnoreCase);
            str = Regex.Replace(str, "the", "da", RegexOptions.IgnoreCase);
            str = Regex.Replace(str, "with", "wif", RegexOptions.IgnoreCase);
            str = Regex.Replace(str, "this", "dis", RegexOptions.IgnoreCase);
            str = Regex.Replace(str, ":D", "UwU", RegexOptions.IgnoreCase);
            str = Regex.Replace(str, ":o", "owo", RegexOptions.IgnoreCase);
            str = Regex.Replace(str, ":O", "OwO", RegexOptions.IgnoreCase);
            str = Regex.Replace(str, @":\\", ":3", RegexOptions.IgnoreCase);
            str = Regex.Replace(str, ":o", "owo", RegexOptions.IgnoreCase);

            str = prefix[rnd.Next(0, prefix.Length)] + " " + str + " " + suffix[rnd.Next(0, suffix.Length)];

            return str;
        }
    }
}

File: Python/owoify.py
# coding=utf-8
import random
import re


def owoify(String: str):
    """
        owoify a given string

        :param String: String to owoify
        :return: owoified string
        """
    _String = String
    prefix = [
        "OWO", "OwO", "0w0", "< 3", "UwU", "HIIII!", "H - hewwo??", "Huohhhh."
    ]
    suffix = [
        "..,", "fwendo", "Huoh.", "._.", ";-;", ";_;", "(；ω；)", "ÙωÙ", "UwU", "(人◕ω◕)", "(●´ω｀●)", "(✿ ♡‿♡)",
        "(◠‿◠✿)", "^ - ^", "^ _ ^", "> _ <", "> _ >", ":P", ":3" ";3", "x3", ":D", "xD", "XDDD", "（＾ｖ＾）",
        "ㅇㅅㅇ", "(• o •)", "ʕ•̫͡•ʔ", "ʕʘ‿ʘʔ", "(　'◟ ')"
    ]

    def repl(m):
        return 'n' + 'u' * (len(m.group()) - 1)

    _String = _String.replace('r', 'w').replace('l', 'w').replace('R', 'W').replace('L', 'W')
    _String = re.sub(r'n[o]+', repl, _String, flags=re.I)
    _String = re.sub(r'ha(ve|s)', 'haz', _String, flags=re.I)
    _String = re.sub('the', 'da', _String, flags=re.I)
    _String = re.sub('with', 'wif', _String, flags=re.I)
    _String = re.sub('this', 'dis', _String, flags=re.I)
    _String = re.sub(':D', 'UwU', _String, flags=re.I)
    _String = re.sub(':o', 'owo', _String, flags=re.I)
    _String = re.sub(':O', 'OwO', _String, flags=re.I)
    _String = re.sub(r':\\', ':3', _String, flags=re.I)
    _String = f"{random.choice(prefix)} {_String} {random.choice(suffix)}"

    return _String
